dashboard/app/controllers/activities_controller.rb

```diff
@@ -15,52 +15,92 @@ class ActivitiesController < ApplicationController
   MIN_LINES_OF_CODE = 0
   MAX_LINES_OF_CODE = 1000
 
+  # Records level progress after each time an attempt is made
+  #
+  # HTTP METHOD: POST only
+  #
+  # GET Params
```

Comment:
Ahh right. Do we have a standard for ruby documentation?  If not I think http://yardoc.org/features.html looks like a good way to go.